lib/team_api/team_member_schema.json

```diff
@@ -0,0 +1,84 @@
+{
+  "team": {
+    "title": "team - [team.member].yml files",
+    "description": "Team member metadata",
+    "$schema": "http://json-schema.org/draft-04/hyper-schema",
+    "type": "object",
+    "properties": {
+      "name": {
```

Comment:
+1 to this; we should do this in a separate PR, and make sure to coordinate the change with anywhere else it might be needed.